roles/ceph-grafana/tasks/configure_grafana.yml

```diff
@@ -46,6 +47,7 @@
     src: dashboards-ceph-dashboard.yml.j2
     dest: /etc/grafana/provisioning/dashboards/ceph-dashboard.yml
     mode: 0640
+  when: not containerized_deployment
```

Comment:
could you add the bool filter there ? ( `| bool` )

Reply:
done